app/models/common/MembersSummary.scala

```diff
@@ -30,15 +32,15 @@ object MembersSummary {
     (JsPath \ "membersDetails" \ "nino").readNullable[String]
       .map(_.getOrElse(messages("site.notEntered")))
 
-  val readsMemberValue: Reads[BigDecimal] =
-    (JsPath \ "totalPensionAmounts").readNullable[BigDecimal]
+  def readsMemberValue(eventType: EventType): Reads[BigDecimal] =
+    memberValuePath(eventType).readNullable[BigDecimal]
       .map(_.getOrElse(BigDecimal(0)))
 
-  private def readsMemberSummary(implicit messages: Messages): Reads[MembersSummary] =
+  private def readsMemberSummary(eventType: EventType)(implicit messages: Messages): Reads[MembersSummary] =
```

Comment:
@grantrobson not quite sure if I've understood this comment - this is the readsMemberSummary method so I'm not sure where I would be nesting this so that eT didn't need to be a parameter.

Reply:
Resolved as per conversation